config/zones.json

```diff
@@ -1991,13 +1991,13 @@
     "capacity": {
       "biomass": 27,
       "coal": 4840,
-      "gas": 12776,
+      "gas": 11010,
```

Comment:
```suggestion
      "gas": 12143.00,
```

Reply:
Based on the active Gas power plants at Israel https://globalenergymonitor.org/projects/global-gas-plant-tracker/tracker-map/